### ISSUE TITLE
Loading view does not animate in WebKit/Safari

### ISSUE THREAD

**Issue by Mr0grog** [edgi-govdata-archiving/web-monitoring-ui#118]
SVG does not animate when used as a background in WebKit, so the loading indicator doesn’t look so great there. The easiest solution here is probably just to render the SVG inline instead of as a background image.

**Comment by Mr0grog** [comment]
> Whoops, sorry about that. I did it as a background image because I had problems understanding how to bundle the image file with the Loading component. I will try again!

No worries! This was still a big improvement over nothing at all and now we’ll make it better :)

The easiest approach might be to use an `<object>` element instead of inlining the SVG:

```html
<object type="image/svg+xml" data="/img/infinity-loader.svg"></object>
```

**Comment by Mr0grog** [comment]
And if you want to inline it, it may be a good time to revisit #88. There are Browserify plugins to do the inlining, but Webpack also supports it. Since we plan to move to Webpack anyway, it’s probably better to do that than add a new Browserify-based tool.

**Comment by lazywithclass** [comment]
I just saw this comment, I've opened https://github.com/edgi-govdata-archiving/web-monitoring-ui/pull/122 but I am happy to revisit it using `object` over `img`, just let me know and I will go for the option that's best practice.

**Comment by Mr0grog** [comment]
This should have been closed in c5a9db7293e1132113ab8604b665d380d88863b7
